ee/clickhouse/views/cohort.py

```diff
@@ -52,17 +52,17 @@ def insert_cohort_actors_into_ch(cohort: Cohort, filter_data: Dict):
     insight_type = filter_data.get("insight")
 
     if insight_type == INSIGHT_TRENDS:
-        filter = Filter(data=filter_data)
+        filter = Filter(data=filter_data).simplify(team=cohort.team)
```

Comment:
Why do we need to call it this way?

The expected behavior is that passing team arg should automatically simplify - callsites don't need to worry beyond this and should not be calling simplify themselves.

Reply:
Hmm was wondering what the pattern here should be. Filter auto simplifies but all the other ones don't so I just did a manual call for uniformity

Reply:
@macobo should I add simplification logic in all the filters? Or was there a reason not to

Reply:
Please pass `team=team` to all filters and _don't_ call simplify outside of tests. This will do the right thing.

Reply:
ah nvm i see where it's simplifying. done!